extmod/asyncio/stream.py

```diff
@@ -100,7 +103,7 @@ def drain(self):
 # Create a TCP stream connection to a remote host
 #
 # async
-def open_connection(host, port):
+async def open_connection(host, port, ssl=None, server_hostname=None):
```

Comment:
This `async` keyword should not be here, because this function only calls `yield`.